packages/devtools_app/lib/src/screens/memory/panes/chart/memory_charts.dart

```diff
@@ -291,39 +291,39 @@ class ChartsValues {
 
     final rssValueDisplay = formatNumeric(vmData[rssJsonName] as num?);
     vmDataDisplayed['$rssDisplay $rssValueDisplay'] = traceRender(
-      color: traces[vm.TraceName.rSS.index].characteristics.color,
+      color: traces[vm.VmTraceName.rSS.index].characteristics.color,
```

Comment:
looks like we still have some prefixes to remove. Let's get rid of the 'events', 'android', and 'vm' prefixes.